src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/CustomRealtimeBlurView.cs

```diff
@@ -0,0 +1,428 @@
+			//mDownsampleFactor = a.getFloat(R.styleable.RealtimeBlurView_realtimeDownsampleFactor, 4);
+
+			mPaint = new Paint();
+		}
+
+		public void SetOverlayColor(int color, bool invalidate = true)
+		{
+			if (mOverlayColor != color)
+			{
+				mOverlayColor = color;
+				if (invalidate)
+				{
+					Invalidate();
+				}
+			}
+		}
+
+		protected IBlurImpl GetBlurImpl()
+		{
+			if (BLUR_IMPL == 0)
+			{
+				// try to use stock impl first
+				if (Build.VERSION.SdkInt >= BuildVersionCodes.JellyBeanMr1)
+				{
+					try
+					{
+						AndroidStockBlurImpl impl = new AndroidStockBlurImpl();
+						Bitmap bmp = Bitmap.CreateBitmap(4, 4, Bitmap.Config.Argb8888);
+						impl.Prepare(Context, bmp, 4);
+						impl.Release();
+						bmp.Recycle();
+						BLUR_IMPL = 3;
+					}
+					catch (Exception e)
+					{
+						if (this.Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Error))
+						{
+							this.Log().Error("Failed GetBlurImpl", e);
+						}
+					}
+				}
+			}
+
+			if (BLUR_IMPL == 0)
+			{
+				// fallback to empty impl, which doesn't have blur effect
+				BLUR_IMPL = -1;
+			}
+			switch (BLUR_IMPL)
+			{
+				case 3:
+					return new AndroidStockBlurImpl();
+				default:
+					return new EmptyBlurImpl();
+			}
+		}
+
+		private class PreDrawListener : Java.Lang.Object, ViewTreeObserver.IOnPreDrawListener
+		{
+			private readonly JniWeakReference<RealtimeBlurView> _weakBlurView;
+
+			public PreDrawListener(RealtimeBlurView blurView)
+			{
+				_weakBlurView = new JniWeakReference<RealtimeBlurView>(blurView);
+			}
+
+			public PreDrawListener(IntPtr handle, JniHandleOwnership transfer)
+				: base(handle, transfer)
+			{
+			}
+
+			public bool OnPreDraw()
+			{
+				if (!_weakBlurView.TryGetTarget(out var blurView))
+				{
+					return false;
+				}
+
+				int[] locations = new int[2];
+				Bitmap oldBmp = blurView.mBlurredBitmap;
+				View decor = blurView.mDecorView;
+				if (decor != null && blurView.IsShown && blurView.Prepare())
+				{
+					bool redrawBitmap = blurView.mBlurredBitmap != oldBmp;
+					oldBmp = null;
+					decor.GetLocationOnScreen(locations);
+					int x = -locations[0];
+					int y = -locations[1];
+
+					blurView.GetLocationOnScreen(locations);
+					x += locations[0];
+					y += locations[1];
+
+					// just erase transparent
+					blurView.mBitmapToBlur.EraseColor(blurView.mOverlayColor & 0xffffff);
+
+					int rc = blurView.mBlurringCanvas.Save();
+					blurView.mIsRendering = true;
+					RENDERING_COUNT++;
+					try
+					{
+						blurView.mBlurringCanvas.Scale(
+							1f * blurView.mBitmapToBlur.Width / blurView.Width,
+							1f * blurView.mBitmapToBlur.Height / blurView.Height);
+						blurView.mBlurringCanvas.Translate(-x, -y);
+						if (decor.Background != null)
+						{
+							decor.Background.Draw(blurView.mBlurringCanvas);
+						}
+						decor.Draw(blurView.mBlurringCanvas);
+					}
+					catch (StopException)
+					{
```

Comment:
Codacy found an issue: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)